bin/spack.bat

```diff
@@ -188,25 +188,28 @@ if NOT "%_sp_args%"=="%_sp_args:--help=%" (
 goto :end_switch
 
 :case_load
-:: If args contain --sh, --csh, or -h/--help: just execute.
-if defined _sp_args (
-    if NOT "%_sp_args%"=="%_sp_args:--help=%" (
-        goto :default_case
-    ) else if NOT "%_sp_args%"=="%_sp_args:-h=%" (
-        goto :default_case
-    ) else if NOT "%_sp_args%"=="%_sp_args:--bat=%" (
-        goto :default_case
-    )
+if NOT defined _sp_args (
+   python "%spack%" "%_sp_subcommand%" --help
```

Comment:
This is not the behavior I observe on linux when `spack load` is invoked with no arguments. There, the command seems to exit silently without invoking the help. Seems like these should be the same (and IMO this approach is better)
@scheibelp thoughts?

Reply:
@scheibelp Can we merge this as is, or should we make it consistent with linux?

Reply:
@johnwparent I removed the help print out, so now the behavior is the same as in Linux. If we want to change that we can do it in a separate commit.